lib/_emerge/Scheduler.py

```diff
@@ -865,13 +865,19 @@ async def _run_pkg_pretend(self, loop=None):
             if self._terminated_tasks:
                 raise asyncio.CancelledError
 
-            out_str = "Running pre-merge checks for " + colorize("INFORM", x.cpv)
-            self._status_msg(out_str)
-
             root_config = x.root_config
             settings = self._allocate_config(root_config.root)
             settings.setcpv(x)
-            if not x.built:
+
+            if x.built:
+                self._status_msg(
+                    f"Running pre-merge checks for {colorize('PKG_BINARY_MERGE', x.cpv)}"
+                )
+            else:
+                self._status_msg(
+                    f"Running pre-merge checks for {colorize('INFORM', x.cpv)}"
+                )
```

Comment:
I'm not saying this is wrong but I'd just make it something like `"INFORM" if x.built else "PKG_BINARY_MERGE"` above to avoid repeating the whole message.

Reply:
sorry - tbh, I'd completely forgot Python allowed that, I'm used to it in Perl. Agreed.